src/client/app/dashboard/patient-dashboard.component.css

```diff
@@ -1,4 +1,5 @@
 @import url(https://fonts.googleapis.com/css?family=Lato:400,700);
+@import url(/node_modules/angular-vertical-timeline/dist/vertical-timeline.css);
```

Comment:
This is the reason for not loading the timeline in deployment.

Reply:
okay